### PR TITLE
Fix relative paths in web_package

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: internal/web_package/injector.js [hunk 1]
```diff
@@ -51,18 +63,33 @@ function main(params, read = fs.readFileSync, write = fs.writeFileSync, timestam
     throw ('No <head> tag found in HTML document');
   }
 
+  /**
+   * Trims the longest prefix from the path
+   */
+  function relative(execPath) {
```
Comment (mgechev) [inline internal/web_package/injector.js]: Node also exposes `relative` in the `path` module.

File: internal/web_package/injector.js [hunk 2]
```diff
@@ -39,6 +39,18 @@ function findElementByName(d, name) {
 function main(params, read = fs.readFileSync, write = fs.writeFileSync, timestamp = Date.now) {
   const outputFile = params.shift();
   const inputFile = params.shift();
+  const rootDirs = [];
+  while (params.length && params[0] !== '--assets') {
+    let r = params.shift();
+    if (!r.endsWith('/')) {
```
Comment (mgechev) [inline internal/web_package/injector.js]: Maybe we can use `path.sep` which returns the correct separator depending on the OS.

Comment (alexeagle) [reply internal/web_package/injector.js]: these paths are constructed by bazel so they should be posix paths always

File: internal/web_package/injector.js (hunk 1 above)
Comment (alexeagle) [reply internal/web_package/injector.js]: yeah, and it has weird cross-platform behavior too

**Pull request merged**